src/core/vote_accounts.zig

```diff
@@ -341,12 +341,12 @@ pub const VoteAccount = struct {
 
         const account = try createVoteAccount(
             allocator,
-            node_pubkey orelse Pubkey.initRandom(random),
-            Pubkey.initRandom(random),
-            Pubkey.initRandom(random),
+            node_pubkey orelse .initRandom(random),
+            .initRandom(random),
+            .initRandom(random),
             random.int(u8),
             random.intRangeAtMost(u64, 1, 1_000_000),
-            Clock.initRandom(random),
+            .initRandom(random),
```

Comment:
I have an ambivalent feelings towards the "dot" syntax. 

I think it prioritize writing over reading code and it may hurt readability. This diff is an instance of such. Previously it is clear what is being initiated...ie `Clock`, `Pubkey` etc but now it is only a bunch of `.initRandom` calls

Reply:
In this case, I agree with trevor, but in the general case, I agree with dadepo. There's definitely information loss which can cause confusion in some cases. But I also appreciate having the option to use this syntax. It makes it a lot easier to *write* code (though readability is more important than writability), plus it tidies things up. Sometimes the lost information is just redundant and pointless, and it's better to reduce the noise.